master/internal/provisioner/scale_decider.go

```diff
@@ -240,3 +204,22 @@ func min(a, b int) int {
 	}
 	return b
 }
+
+func findLongLived(
+	candidate map[string]time.Time, stillLived map[string]bool, duration time.Duration,
```

Comment:
non-blocking: rename `stilllived` to `idleInstances`

Reply:
Talked offline: we decide to use `pastIdleInstances` and `presentIdleInstances`